server/parsec/components/shamir.py

```diff
@@ -0,0 +1,130 @@
+# Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS
+
+from __future__ import annotations
+
+from enum import auto
+
+from parsec._parsec import (
+    DeviceID,
+    OrganizationID,
+    ShamirRecoveryBriefCertificate,
+    ShamirRecoveryShareCertificate,
+    UserID,
+    VerifyKey,
+    authenticated_cmds,
+)
+from parsec.api import api
+from parsec.client_context import AuthenticatedClientContext
+from parsec.types import BadOutcomeEnum
+
+
+class BaseShamirComponent:
+    @api
+    async def create_shared_recovery_device(
```

Comment:
Should be called `api_<name of the command>`, so `api_shamir_recovery_setup` for consistency with other routes